extensions/wikia/Wall/WallRelatedPages.class.php

```diff
@@ -151,6 +158,7 @@ public function getArticlesRelatedMessgesSnippet( $pageId, $messageCount, $reply
 		$helper = new WallHelper();
 
 		foreach ( $messages as $value ) {
+			// TODO: use WallThread::newFromIds to fetch all threads in a single batch
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)